src/components/TextInputFocusable/index.js

```diff
@@ -353,7 +335,7 @@ class TextInputFocusable extends React.Component {
                 onChange={() => {
                     this.updateNumberOfLines();
                 }}
-                onSelectionChange={this.saveSelection}
+                onSelectionChange={this.onSelectionChange}
```

Comment:
Shouldn't this be `this.props.onSelectionChange`? Because `onSelectionChange` does not exist as a method on `Composer`

Reply:
@parasharrajat